src/integrationTest/java/uk/gov/hmcts/reform/managecase/fixtures/WiremockFixtures.java

```diff
@@ -183,6 +187,50 @@ public static void stubIdamGetUserById(String userId, UserDetails user) {
                         .withHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)));
     }
 
+    public static void stubGetStartEventTrigger(String caseId,
+                                                String eventId,
+                                                CaseUpdateViewEvent caseUpdateViewEvent) {
+        stubFor(WireMock.get(urlPathEqualTo("/internal/cases/" + caseId + "/event-triggers/" + eventId))
+                    .withHeader(AUTHORIZATION, equalTo(SYS_USER_TOKEN))
+                    .withHeader(SERVICE_AUTHORIZATION, equalTo(S2S_TOKEN))
+                    .willReturn(aResponse()
+                                    .withStatus(HTTP_OK)
+                                    .withBody(getJsonString(caseUpdateViewEvent))
+                                    .withHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)));
+    }
+
+    public static void stubGetCaseViaExternalApi(String caseId, CaseDetails caseDetails) {
+        stubFor(WireMock.get(urlPathEqualTo("/cases/" + caseId))
+                    .withHeader(AUTHORIZATION, equalTo(SYS_USER_TOKEN))
+                    .withHeader(SERVICE_AUTHORIZATION, equalTo(S2S_TOKEN))
+                    .willReturn(aResponse()
+                                    .withStatus(HTTP_CREATED)
```

Comment:
HTTP_CREATED?

Reply:
Fixed in commit https://github.com/hmcts/aac-manage-case-assignment/pull/150/commits/04ca3c6315a335ded9f9683f351c1dbf78f7d120